docs/naps/4-async-slicing.md

```diff
@@ -511,12 +588,21 @@ it emits the `slice_ready` event. There are a few reasons for that.
 1. We only use one slicing thread to keep behavior simple and to avoid GIL contention.
 2. It's closer to the existing behavior of napari
 3. Shouldn't introduce any new potential bugs, such as [^issue-2862].
-4. It doesn't need any UX design work to decide what should be shown while we are waiting for slices to be ready.
+4. It needs less UX design work to decide what should be shown while we are waiting for slices to be ready.
 
 In some cases, rendering slices as soon as possible will provide a better user experience,
 especially when some layers are substantially slower than others. Therefore, this should be
-high priority future work. One way to implement this behavior is to emit a `slice_ready`
-event per layer that only contains that layer's slice response.
+high priority future work.
+
+To implement this behavior as a small extension to this proposal, we could do something like the following.
```

Comment:
@jni : expanded this to describe what an extension could look like to support this with a link to a diagram below.